### PR TITLE
Make the `Plugin folder` notification user friendly

### LINKED ISSUE #5693: [hosted-plugin] Make the `Plugin folder` notification user friendly
When I start a hosted instance on Windows, I can see the following as a notification.

Screen Shot 2019-07-11 at 10 36 02

`file:///c%3A/` is not a thing I would like to see as a Windows user. It is alien.

TODO:
 - Use the path instead of the URI, _OR_
 - Decode the URI.

### PR DESCRIPTION
**Pull request by chattarajoy**
Fixes #5693.

The message displayed contained an encoded URI, added changes to decode it before displaying the message.

### CODE REVIEW

**Comment by akosyakov** [comment]
Thank you for the PR. We should use `FileSystem.getFsPath`, see coding guidelines: https://github.com/theia-ide/theia/wiki/Coding-Guidelines#frontend-fs-path

File: packages/plugin-dev/src/browser/hosted-plugin-manager-client.ts [hunk 1]
```diff
@@ -265,7 +265,7 @@ export class HostedPluginManagerClient {
         if (UriSelection.is(result)) {
             if (await this.hostedPluginServer.isPluginValid(result.uri.toString())) {
                 this.pluginLocation = result.uri;
-                this.messageService.info('Plugin folder is set to: ' + result.uri.toString());
+                this.messageService.info('Plugin folder is set to: ' + decodeURI(result.uri.toString()));
```
Comment (kittaakos) [inline packages/plugin-dev/src/browser/hosted-plugin-manager-client.ts]: You can use ` result.uri.toString(true)` instead.

Comment (akosyakov) [reply packages/plugin-dev/src/browser/hosted-plugin-manager-client.ts]: thinking again, `LabelProvider.getLongName` should be used to a display string from a URI to align with the rest of UI

**Comment by kittaakos** [comment]
@chattarajoy, please squash your commits.

**Comment by chattarajoy** [comment]
@kittaakos, have squashed the commits

**Comment by kittaakos** [comment]
> @kittaakos, have squashed the commits

I am trying this now on Windows.

**Comment by kittaakos** [comment]
It looks better now on Windows. Thank you for the help, @chattarajoy!

Screen Shot 2019-07-18 at 16 24 01

**Pull request merged**